### ISSUE TITLE
CampaignMonitor image height compatibility

### ISSUE THREAD

**Issue by kmedeiros** [emailmonday/Cerberus#80]
Hi Ted, 

I noticed that CampaignMonitor's email builder automatically applies a height to an uploaded image even though the height is not present in the uploaded HTML, causing the image to stretch vertically in the default email client on iPhone 7.

The editable image in the CampaignMonitor template looks like this:

```
<img editable src="http://placehold.it/270x152/dddddd/666666.jpg" width="270" label="Image" alt="Image description" style="border:0;width:100%;max-width:270px;height:auto;background:#dddddd;font-family:sans-serif;font-size:15px;mso-height-rule:exactly;line-height:20px;color:#555555;">
```

but then the height property is added to the <img> tag in the HTML output:

```
<img editable src="http://www.userdomain.com/uploadedimage.jpg" width="270" height="152" label="Image" alt="Image description" style="border:0;width:100%;max-width:270px;height:auto;background:#dddddd;font-family:sans-serif;font-size:15px;mso-height-rule:exactly;line-height:20px;color:#555555;">
```

I simply added `height: auto;` to the `img` element on line 72 and the problem went away. I tested it in Litmus.

**Comment by TedGoas** [comment]
@kmedeiros If Campaign Monitor is modifying the code, that's not cool! Glad you found a simple and reliable fix, though!

Since there's nothing inherently wrong with the code in this repo, I'm not sure there's anything to _fix_, right? The problem is with Campaign Monitor's email builder.

**Comment by kmedeiros** [comment]
You're correct, there's nothing wrong with the code. Just thought it might be safe to include `height: auto;` to all images by default, since there may be other ESPs out there that apply a fixed height to images  when users generate templates through their system. I ran into the same problem with a system called iModules Encompass. It's definitely not cool when any system decides to override your custom code but it happens.

**Comment by TedGoas** [comment]
@kmedeiros Ahh good point. I can make this update and test. Can't imagine this would cause issues in any other client, but will report back if it does.

**Comment by TedGoas** [comment]
@kmedeiros Thanks again, I pushed an update that should cover this... with one caveat.

For some reason, the thumbnails in `cerberus-fluid.html` became distorted in Outlook 2000/2002 when I applied `height: auto;`. I tried a number of fixes, including pasting a working image from another template. Weird, but can be worked around.

![ol2000-vertical-allowed-1366](https://cloud.githubusercontent.com/assets/1172461/19543298/75f9de9e-9644-11e6-97b8-86a0f757180b.png)

**Comment by kmedeiros** [comment]
Just an update on this...

During a recent test on Gmail (web) and G Suite I noticed that the images are distorted like you mentioned above. It looks like it wasn't as reliable as I thought.

I found by adding `height: auto;` it caused problems when attempting to use images @2x for retina screens. For example, if using an image at `400x400` pixels for a container that is set to `200x200` pixels. 

I had to replace `height: auto;` with either `max-height: 200px;` or `height: 200px;`

When modifying Cerberus to be used in the template wizard within CampaignMonitor, be aware that you still have to add `height: auto;` to the `<img editable>` tag if you want the image to scale vertically. https://www.campaignmonitor.com/create/editable-content/#image
